src/snowflake/snowpark/mock/mock_select_statement.py

```diff
@@ -62,17 +108,19 @@ def __init__(
         self.post_actions = None
         self.flatten_disabled: bool = False
         self._column_states: Optional[ColumnStateDict] = None
-        self.expr_to_alias = {}
         self._execution_plan: Optional[SnowflakePlan] = None
         self._attributes = None
 
     @property
     def execution_plan(self):
+        """Convert to a SnowflakePlan"""
+        if self._execution_plan is None:
+            self._execution_plan = MockExecutionPlan(self, self.analyzer.session)
```

Comment:
is this the reason for the circular import?
I'm wondering what's the case that `self._execution_plan` could be passed as None and we need to wrap it here.